src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.ts

```diff
@@ -84,10 +85,12 @@ export class InstructorHomePageComponent extends InstructorSessionModalPageCompo
               studentService: StudentService,
               instructorService: InstructorService,
               tableComparatorService: TableComparatorService,
-              private simpleModalService: SimpleModalService,
+              simpleModalService: SimpleModalService,
```

Comment:
I think we should keep services private

Reply:
We cannot set the access modifier to `private` here. `InstructorHomePageComponent` inherits from `InstructorSessionModalPageComponent` , which in turn inherits from `InstructorSessionBasePageComponent`, so this imposes some limitations for the access modifiers of the declared services.

For example, if we change the access modifier to private, we'll get an error like this:
<img src="https://user-images.githubusercontent.com/46853051/112731573-1ddd8580-8f73-11eb-8b5f-4f8fefe438fa.png" width="400"/>

If we change the access modifier in the top level class to private as well, we get another error:
<img src="https://user-images.githubusercontent.com/46853051/112731558-0c947900-8f73-11eb-80db-5b3d16f7415e.png" width="400"/>

Reply:
I'm not too sure about this error, but the original version has `private`. From the error messages, it seems like the parent component has `public` instead of private, maybe we can change that

Reply:
Please refer to the parent class `instructor-session-base-page.component.ts`. `simpleModalService` has been declared in the parent class (instead of the child). With this change, it is no longer possible to keep the `private` access modifier in the child class.
![image](https://user-images.githubusercontent.com/46853051/112747438-0e4c5400-8fe8-11eb-851b-fee20e0959e2.png)

Notice that: If the service is declared in the parent class with an access modifier, then the child class should **not** declare the service with an access modifier (including `private`). From my understanding, the reason is that:
- Without an access modifier, the constructor parameter acts as a normal method parameter. 
- With an access modifier, the parameter is automatically promoted to be a class property. In other words, the access modifier tells Angular to create a property of the same name within the class. If the parent class contains this property, then the child class inherits the property and cannot re-declare this property. Adding an access modifier implicitly re-declares this property, so it is not allowed here.

Reply:
Oh ok I see now. Then its ok to leave as it is